src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -494,8 +496,7 @@ else if (message.isDDLMessage()) {
                                 }
                             } catch (InterruptedException ie) {
                                 ie.printStackTrace();
-                            } catch (SQLException se) {
```

Comment:
Is the sqlexception catch removed? Can you explain why?

Reply:
This is no longer needed since we will not throw a 'SQLException' from any part of the codec within the try.